gradebookng/tool/src/webapp/styles/gradebook-grades.css

```diff
@@ -705,12 +705,9 @@ div.wicket-modal div.w_right > div {
 div.wicket-modal {
   z-index: 999 !important;
   max-width: 100%;
+  position: absolute;
```

Comment:
Is this absolute positioning needed? I see that Wicket applies this to the modal inline already.

Reply:
Yes, it's needed to make Safari work properly. Without it, you get this:
<img width="718" alt="screen shot 2019-01-16 at 09 33 28" src="https://user-images.githubusercontent.com/134546/51239991-b2e79b80-1972-11e9-9c77-9e8376ba520a.png">
Hidden content and no scrolling of the modal into view.

With it you get this:
<img width="739" alt="screen shot 2019-01-16 at 09 36 29" src="https://user-images.githubusercontent.com/134546/51240026-c98df280-1972-11e9-8b2d-5baecea6e1ea.png">
You can scroll the modal into view and see the buttons.

Reply:
Okay. Thanks for confirming. 

What you have now looks good to me then.

Reply:
@fostersdesign Could you review, please? I can merge it then.

Reply:
I have already changed my review to "Approved". I just don't have any authority here, so it doesn't count as a required review.

Maybe ask @ern or @steveswinsburg for their approval too?

Reply:
@ern Any chance?